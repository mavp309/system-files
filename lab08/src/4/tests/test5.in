-0.199 -4.5336 1.41 33.33
