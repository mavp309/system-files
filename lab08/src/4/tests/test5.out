16.4660 -3.8286
